python/pyspark/pandas/indexes/timedelta.py

```diff
@@ -111,3 +115,66 @@ def __getattr__(self, item: str) -> Any:
                 return partial(property_or_func, self)
 
         raise AttributeError("'TimedeltaIndex' object has no attribute '{}'".format(item))
+
+    @property
+    def days(self) -> Index:
+        """
+        Number of days for each element.
+        """
+        sdf = self._internal.spark_frame.select(
```

Comment:
Hm .. shouldn't we use `self.spark.column` or with `_with_new_scol` instead of doing a select? Otherwise, the output will have a different anchor.

Reply:
Makes sense, let me adjust that, thanks!

Reply:
Considering the negative time delta, `Series.apply` is used instead.